src/main/java/com/minecolonies/coremod/entity/ai/citizen/guard/DruidCombatAI.java

```diff
@@ -0,0 +1,340 @@
+        }
+
+        this.instantEffect = effect.isInstantenous();
+
+        user.setItemInHand(Hand.MAIN_HAND, stack);
+
+        user.getThreatTable().removeCurrentTarget();
+
+        user.decreaseSaturationForContinuousAction();
+        user.getCitizenExperienceHandler().addExperience(PER_POTION_XP);
+    }
+
+    @Override
+    protected int getAttackDelay()
+    {
+        return this.instantEffect ? super.getAttackDelay() * 2 : super.getAttackDelay();
+    }
+
+    @Override
+    protected double getAttackDistance()
+    {
+        int attackDist = BASE_DISTANCE_FOR_POTION_ATTACK;
+        // + 1 Blockrange per building level for a total of +5 from building level
+        if (user.getCitizenData().getWorkBuilding() != null)
+        {
+            attackDist += user.getCitizenData().getWorkBuilding().getBuildingLevel();
+        }
+
+        if (target != null)
+        {
+            attackDist += user.getY() - target.getY();
+        }
+
+        return attackDist;
+    }
+
+    @Override
+    protected PathResult moveInAttackPosition(final LivingEntity target)
+    {
+        if (BlockPosUtil.getDistanceSquared(target.blockPosition(), user.blockPosition()) <= 4.0)
+        {
+            final PathJobMoveAwayFromLocation job = new PathJobMoveAwayFromLocation(user.level,
+              AbstractPathJob.prepareStart(target),
+              target.blockPosition(),
+              12,
+              (int) user.getAttribute(Attributes.FOLLOW_RANGE).getValue(),
+              user);
+            final PathResult pathResult = ((MinecoloniesAdvancedPathNavigate) user.getNavigation()).setPathJob(job, null, getCombatMovementSpeed(), true);
+            job.setPathingOptions(combatPathingOptions);
+            return pathResult;
+        }
+        else if (BlockPosUtil.getDistance2D(target.blockPosition(), user.blockPosition()) >= 20)
+        {
+            final PathJobMoveToLocation job = new PathJobMoveToLocation(user.level, AbstractPathJob.prepareStart(user), target.blockPosition(), 200, user);
+            final PathResult pathResult = ((MinecoloniesAdvancedPathNavigate) user.getNavigation()).setPathJob(job, null, getCombatMovementSpeed(), true);
+            job.setPathingOptions(combatPathingOptions);
+            return pathResult;
+        }
+        final PathJobCanSee job = new PathJobCanSee(user, target, user.level, ((AbstractBuildingGuards) user.getCitizenData().getWorkBuilding()).getGuardPos(), 40);
+        final PathResult pathResult = ((MinecoloniesAdvancedPathNavigate) user.getNavigation()).setPathJob(job, null, getCombatMovementSpeed(), true);
+        job.setPathingOptions(combatPathingOptions);
+        return pathResult;
+    }
+
+    /**
+     * Get combat speed
+     *
+     * @return movent speed
+     */
+    protected double getCombatMovementSpeed()
+    {
+        double levelAdjustment = user.getCitizenData().getCitizenSkillHandler().getLevel(Skill.Mana) * SPEED_LEVEL_BONUS;
```

Comment:
The druid should probably be less agile than a ranger and get less of a speed bonus. Also a bit weird to get a speed bonus from the mana stat

Reply:
Reduced slightly. Their other stat is focus, so it's even less fitting. Both knights and archers got this one